src/modelo/Gnar.java

```diff
@@ -2,7 +2,7 @@
 
 public class Gnar extends Starter {
 
-    private int resistencia;
+    private int resistencia; //PARA FICHEROS
```

Comment:
Is this comment relevant? What about expressing this intention in the variable name instead?

Reply:
It isn't. I'll change it